server/channels/app/user_test.go

```diff
@@ -1119,7 +1153,6 @@ func TestPermanentDeleteUser(t *testing.T) {
 
 	createdScheduledPost2, appErr := th.App.SaveScheduledPost(th.Context, scheduledPost2, "")
 	require.Nil(t, appErr)
-
```

Comment:
Unrelated change?

Reply:
Not sure why this changed - Should I add a blank line here?